arxiv-opensearch-cohere/README.md

```diff
@@ -0,0 +1,256 @@
+# Cohere / Opensearch implementation
+
+The following steps outline how a user would perform vector search with Cohere and Opensearch. Starting with spinning up a local OpenSearch instance and how a user would store Cohere embeddings in an OpenSearch index, and then explaining the methods for retrieval using these embeddings.
```

Comment:
nit OpenSearch is right, couple places including line 1 use Opensearch

Reply:
good catch, fixed!